templates/CRM/Contribute/Page/Tab.tpl

```diff
@@ -31,18 +31,8 @@
     <div class="contact-summary-contribute-tab view-content">
 
       <div id="secondaryTabContainer" class="ui-tabs ui-widget ui-widget-content ui-corner-all">
-        {* Tab management *}
-        <script type="text/javascript">
-          var selectedTab  = 'contributions';
+        {include file="CRM/common/TabSelected.tpl" defaultTab="contributions" tabContainer="#secondaryTabContainer"}
```

Comment:
I couldn't figure out how to select the recurring subtab (before or after this change)

Reply:
@eileenmcnaughton No, you can't :-(  But maybe once this abstraction is in place we might be able to look at it